clients/src/main/java/org/apache/kafka/common/requests/ApiVersionsResponse.java

```diff
@@ -258,11 +259,22 @@ private static ApiVersionsResponseData createApiVersionsResponseData(
         final long finalizedFeaturesEpoch,
         final boolean zkMigrationEnabled
     ) {
+        Features<SupportedVersionRange> backwardsCompatibleFeatures = Features.supportedFeatures(latestSupportedFeatures.features().entrySet()
+            .stream().filter(entry -> {
+                SupportedVersionRange supportedVersionRange = entry.getValue();
+                return supportedVersionRange.min() != 0;
```

Comment:
not sure whether I fail to catch your description.

> so for now we will change 0 to 1 in the response in order to be backwards compatible.

 the code looks like it gets rid of "0" instead of changing to from 0 to 1.

Reply:
For another, does this PR mean `Admin#describeFeatures` can't see the feature "group.version" from the broker running in 3.8.0?

Reply:
Hey sorry, the comment was from the old implementation, the new way is to omit as you mentioned.
I believe describeFeatures will not be able to show the version for this release.